_posts/2023-05-15-gsoc.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "I Got Into GSoC!"
+date: 2023-05-15 09:24
+category: kernel developing
+tags: GSoC open-source graphics
+---
+
+Wow, it's really happening :D
+
+After some [warm-up][warm-up] and work put on, I got accepted into the 2023
+Google Summer of Code program with the X.Org organization.
+
+# What I'm Going to Do
+
+The title of my project is Increasing Code Coverage on the DRM Code. I will
+create unit tests for the drm-*-helper, specifically, the drm-format-helper.c. I
```

Comment:
You might want to format some technical names. Can you post a screenshot of the rendered result? (I'm assuming there's no preview for this)

```suggestion
create unit tests for the `drm-*-helper`, specifically, the drm-format-helper.c. I
```

Reply:
![image](https://github.com/Grillo-0/blog/assets/53825439/96f27b65-7c46-4aa0-be93-c9687305bdf0)

Yeah, it becomes much better :)